client/commands/zeth

```diff
@@ -34,9 +35,11 @@ def help(ctx: Context) -> None:
 
 @group(cls=DefaultGroup, default_if_no_args=True, default="help")
 @option(
-    "--eth-rpc",
-    default=ETH_RPC_ENDPOINT_DEFAULT,
-    help=f"Ethereum rpc end-point")
+    "--eth-network",
+    default=None,
+    help="Ethereum RPC endpoint, network or config file "
```

Comment:
Wouldn't it be possible to stick to one type of values here? I mean: do we really need the 3 potential value types?
I find it confusing to see that a flag can take 3 type of values (a URL or a "name" or a file).

Reply:
I definitely get the point.  I'm sure it can be improved with more time invested, but my thinking was that each of these forms is a potential use-case, and there should not really be any ambiguity when specifying this.  I considered having multiple flags to cover different types, but that would involve quite a bit more complexity and logic to resolve some ambiguous cases like multiple flags being given, overriding each other and falling though to defaults, etc.

So this seemed like a simple solution that avoids ambiguity, but definitely happy to iterate if there is a better way to expose the same functionality.

Reply:
I see, I'll open a ticket to keep this in mind and polish this going forward.

Reply:
Opened https://github.com/clearmatics/zeth/issues/249 to track this